app/config/collections.php

```diff
@@ -228,7 +228,7 @@
             [
                 '$collection' => Database::SYSTEM_COLLECTION_RULES,
                 'label' => 'Password Update Date',
-                'key' => 'password-update',
+                'key' => 'passwordUpdate',
```

Comment:
Thank you, you don't have any idea how much that bothered me hehe 🤪

Reply:
Hehe 👏🏻